src/components/Dashboard.tsx

```diff
@@ -85,9 +94,10 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{ template: TemplateDetails; filter: Filter }> = ({
+const FilterComponent: React.FC<{ template: TemplateDetails; filter: Filter; setLoading:any }> = ({
```

Comment:
Try not to use `any` unless it's really necessary.

In this case the correct type for `setLoading` is `(loading: boolean) => void`